partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -0,0 +1,77 @@
+#
+
+
+import rich_click as click
+import partcad.utils as pc_utils
+from partcad import logging as logging
+
+import argparse
+import asyncio
+
+import partcad.logging as pc_logging
+import partcad.utils as pc_utils
```

Ruff (0.8.2)

Redefinition of unused `pc_utils` from line 12

Remove definition: `pc_utils`

(F811)